gensim/models/word2vec.py

```diff
@@ -746,6 +753,9 @@ def job_producer():
             job_batch, batch_size = [], 0
             pushed_words, pushed_examples = 0, 0
             next_alpha = self.alpha
+            if next_alpha > self.min_alpha_yet_reached:
```

Comment:
yes I was able to trigger it by changing the alpha manually like:
```python
model = Doc2Vec(alpha=0.025, min_alpha=0.025, min_count=1, workers=8, size=5)

model.build_vocab(tagged_docs)

for epoch in range(10):

    model.train(tagged_docs)

    model.alpha -= 0.002

    model.min_alpha = model.alpha

    if epoch == 5:
      model.alpha += 0.05
```
The user won't exactly do this but I just wanted to highlight that manual updation can trigger this.
Yes my intent was to make the user aware that the alpha, which should be monotonically decreasing over training cycles, has increased. Yep rewording it like that would be better.